Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
It gives a grunt not found error. Then manually if I installed the supposedly missing grunt package, there'd be another package it would claim is missing, and another and another ...